test/classes/Server/PrivilegesTest.php

```diff
@@ -1840,7 +1840,7 @@ public function testGetFormForChangePassword(): void
 
         //labels
         $this->assertStringContainsString(
-            __('Change password'),
+            __('change_password_form'),
```

Comment:
```suggestion
            'change_password_form',
```
It has no translation ;)

Reply:
Yeah 😅